guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
+
 Published in January 2021 by Artemij Voskobojnikov, Oliver Wiese, Masoud Mehrabi Koushki, Volker Roth, and Konstantin Beznosov.
 
 ###### [User Perception of Bitcoin Usability and Security across Novice Users](https://www.researchgate.net/publication/331286661_User_Perception_of_Bitcoin_Usability_and_Security_across_Novice_Users)
+
+*This paper investigates users’ perceptions and experiences of an anonymous digital payment system (Bitcoin) and its influence on users in terms of usability and security in comparison to other non-anonymous payment systems such as credit/debit cards. This paper identifies users’ perceptual differences in terms of usability and security. We conclude that Bitcoin, as a crypto-currency, is still in its infancy and requires user education and a new way of thinking. The study recommends developing users’ mental models to deepen developers’ understanding of anonymous digital payment technology and improve user-centred design.*
+
 Published in February 2019 by Abdulla Alshamsi & Keele University.
 
 ###### [A First Look at the Usability of Bitcoin Key Management](https://arxiv.org/pdf/1802.04351.pdf)
+
+*Bitcoin users are directly or indirectly forced to deal with public key cryptography, which has a number of security and usability challenges that differ from the password-based authentication underlying most online banking services. In this paper, we contribute an evaluation framework for comparing Bitcoin key management approaches, and conduct a broad usability evaluation of six representative Bitcoin clients. We find that Bitcoin shares many of the fundamental challenges of key management known from other domains, but that Bitcoin may present a unique opportunity to rethink key management for end users.*
+
 Published in February 2018 by Shayan Eskandari, David Barrera, Elizabeth Stobert, and Jeremy Clark.
 
 ###### [An Evaluation of User Attitudes Towards Anonymity in Bitcoin](https://www.diva-portal.org/smash/get/diva2:1141672/FULLTEXT01.pdf)
+
+*The research effort presented in this paper consists of a quantitative study seeking to address the subject of user anonymity in the Bitcoin network. The survey seeks to form an understanding of user attitudes towards the aspect of anonymity by following a methodological approach for exploring common tendencies among the representatives and will serve as the underlying data set from which conclusions can be drawn. Furthermore, this paper will present a literary study of the actual state of anonymity in this peer-to-peer technology by reviewing current findings highlighted in the area, thus presenting a comprehensive view of anonymity in the Bitcoin network.*
```

Comment:
```suggestion
*A survey of 50 users on the [r/bitcoin subreddit](https://www.reddit.com/r/Bitcoin/) about their attitudes towards anonymity in the Bitcoin network.*
```
This was really just a small survey on Reddit, so we don't need this super academic explanation.